src/components/app-rating-mobile/AppRatingMobile.tsx

```diff
@@ -11,16 +11,17 @@ import { TotalReviews, UserRoleEnum } from '~/types'
 
 interface AppRatingMobileProps extends RatingProps {
   reviewsCount: TotalReviews
+  hideStar?: boolean
 }
 
-const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount }) => {
+const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount, hideStar = false }) => {
   const { userRole } = useAppSelector((state) => state.appMain)
   const { t } = useTranslation()
 
   return (
     <Box sx={styles.root}>
       <Box data-testid='number-box' sx={styles.number}>
-        <StarSharp data-testid='star-icon' sx={styles.starMobile} />
+        {!hideStar && <StarSharp data-testid='star-icon' sx={styles.starMobile} />}
```

Comment:
You don't need to change this component. According to Figma, we shouldn't hide the rating.
For more understanding, we use two types of cards: square and rectangular. It can see in Figma.

Reply:
in squares everything like it was before, i used hideStar, because AppRatingMobile with reviewsCount placing star, anyway i will try to do it without changing AppRatingMobile